tests/test_self_containment.cc.in

```diff
@@ -0,0 +1,27 @@
+/**
+ * \file   test_self_containment.cc.in
+ * \author \ref contributors
+ * \date   Created on August 21, 2019
+ * \brief  Used to check if the individual headers are self-contained
+ *
+ * \copyright Copyright 2019 Deutsches Elektronen-Synchrotron (DESY), Hamburg
```

Comment:
... also here 😉

Reply:
Hmm? That file has been create in 2019 and not touched since ...